Build/Blazorise.props

```diff
@@ -61,4 +61,56 @@
 		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="8.0.*" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.*" />
 	</ItemGroup>
+
+	<!--<Target Name="ReplaceVersionPlaceholderInJsFiles" BeforeTargets="Build">
+
+		<PropertyGroup>
+			<Placeholder>\?v=([\d.]+)</Placeholder>
+			<Replacement>?v=$(Version)</Replacement>
+		</PropertyGroup>
+		
+		<Exec Command="PowerShell -Command &quot;if (Test-Path 'wwwroot') { Get-ChildItem -Path 'wwwroot' -Filter *.js | ForEach-Object { (Get-Content $_.FullName) -replace '$(Placeholder)', '$(Replacement)' | Set-Content $_.FullName }}&quot;" />
+	</Target>-->
+
+	<UsingTask
+	  TaskName="ReplaceVersionPlaceholderInJsFiles"
+	  TaskFactory="RoslynCodeTaskFactory"
+	  AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+		<ParameterGroup>
+			<SourceFiles ParameterType="Microsoft.Build.Framework.ITaskItem[]" Required="true" />
+			<MatchExpression ParameterType="System.String" Required="true" />
+			<ReplacementText ParameterType="System.String" Required="true" />
+		</ParameterGroup>
+		<Task>
+			<Using Namespace="System"/>
+			<Using Namespace="System.IO"/>
+			<Using Namespace="System.Text.RegularExpressions" />
+			<Code Type="Fragment" Language="cs">
+				<![CDATA[  
+				Log.LogMessage(MessageImportance.High, $"------ Custom Build Step : ReplaceVersionPlaceholderInJsFiles : [{SourceFiles.Count()}] files ------");
+                foreach (ITaskItem item in SourceFiles)
+                {
+					string fileName = item.ItemSpec;
+					Log.LogMessage(MessageImportance.High, "------ Custom Build Step : ReplaceVersionPlaceholderInJsFiles : Updating File '{0}'.", fileName);
+					
+					File.WriteAllText(
+					fileName,
+					Regex.Replace(File.ReadAllText(fileName), MatchExpression, ReplacementText)
+					);
+				}
+        ]]>
+			</Code>
+		</Task>
+	</UsingTask>
+
+	<Target Name="BeforeBuildStep" BeforeTargets="Build">
+		<ItemGroup>
+			<JsFiles Include='wwwroot\*.js' />
+		</ItemGroup>
+		
+		<ReplaceVersionPlaceholderInJsFiles
+		   SourceFiles="@(JsFiles)"
+		   MatchExpression="\?v=([\d.]+)"
+		   ReplacementText="?v=$(Version).0" />
```

Comment:
What will happen when we have a version that `1.4.3.1`? This will convert it to `1.4.3.1.0`,

Reply:
This is temporary. I need to get proper assembly version like I've already told before.
![image](https://github.com/Megabit/Blazorise/assets/22283161/9d833c04-82c4-4ac5-8d79-b80982e6376d)

Reply:
Actually it might be best to correctly set the version with 4 digit groups in the project if that is expected.
Although it's naive to assume we'll ever do **1.4.3.1** as If I remember correctly we've defined that we would stick to the semantic versioning which just uses the first 3 groups?

Now with automatic versioning. Maybe you could change the JS files to be 3 digit groups instead? Because the original issue where it would be easier for you to replace no longer applies, since it's automatic.

Reply:
We had invalid patches, so I had to use the fourth number to fix it.

![image](https://github.com/Megabit/Blazorise/assets/900302/3a7d68e4-5357-4cfb-b912-dff7412345bd)

Reply:
I see. Well if everything works as expected, you just need to properly set the Version. And it will propagate properly. See my latest comment for results.